stock_dynamic_routing/models/stock_move.py

```diff
@@ -564,7 +563,9 @@ def _insert_routing_moves(self, picking_type, location, destination):
         )
         if dest_moves:
             dest_moves.write({"move_orig_ids": [(3, self.id), (4, routing_move.id)]})
-        routing_move._action_confirm(merge=False)
+        routing_move.with_context(stock_dynamic_routing_confirm=True)._action_confirm(
```

Comment:
That one is maybe not required, I blocked the automatic reservation on all `_action_confirm` calls.